sdk/entra/WebJobs.Extensions.AuthenticationEvents/src/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents.csproj

```diff
@@ -0,0 +1,64 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+	<PropertyGroup>
+		<TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+		<IsExtensionClientLibrary>true</IsExtensionClientLibrary>
+		<Version>1.0.0-beta.1</Version>
+		<Description>Triggers for Azure AD Authentication event custom extensions. Lets you focus on your business logic.</Description>
+		<GeneratePackageOnBuild>False</GeneratePackageOnBuild>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<PackageReference Include="Microsoft.Azure.WebJobs" />
+		<PackageReference Include="Microsoft.AspNetCore.Http" />
+		<PackageReference Include="Microsoft.IdentityModel.Protocols" />
+		<PackageReference Include="Microsoft.IdentityModel.Protocols.OpenIdConnect" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Compile Remove="HttpTriggerAttribute.cs" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="Templates\ActionableTemplate.json" />
+		<EmbeddedResource Include="Templates\CloudEventActionableTemplate.json" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<Compile Update="AuthEventResource.Designer.cs">
+		  <DesignTime>True</DesignTime>
+		  <AutoGen>True</AutoGen>
+		  <DependentUpon>AuthEventResource.resx</DependentUpon>
+		</Compile>
+		<Compile Update="AuthEventResourceAuthEventResource.Designer.cs">
+			<DesignTime>True</DesignTime>
+			<AutoGen>True</AutoGen>
+			<DependentUpon>AuthEventResourceAuthEventResource.resx</DependentUpon>
+		</Compile>
+	</ItemGroup>
+
+
+	<ItemGroup>
+		<EmbeddedResource Update="AuthEventResource.resx">
+		  <Generator>ResXFileCodeGenerator</Generator>
+		  <LastGenOutput>AuthEventResource.Designer.cs</LastGenOutput>
+		</EmbeddedResource>
+		<EmbeddedResource Update="AuthEventResourceAuthEventResource.resx">
+			<Generator>ResXFileCodeGenerator</Generator>
+			<LastGenOutput>AuthEventResourceAuthEventResource.Designer.cs</LastGenOutput>
+		</EmbeddedResource>
+	</ItemGroup>
+
+
+	<ItemGroup>
+		<None Update=".gitignore">
```

Comment:
Remove?

Reply:
Apologies, I removed the physical file via file system, so link was not removed. Done.